band/java/src/main/java/org/mrsnu/band/Buffer.java

```diff
@@ -0,0 +1,28 @@
+package org.mrsnu.band;
+
+import android.media.Image.Plane;
+
+public class Buffer {
+  private NativeBufferWrapper wrapper;
+
+  Buffer(final byte[] buffer, int width, int height, BufferFormat bufferFormat) {
```

Comment:
All constructors for this class should be public so that Buffer can be used by JAVA developers through the API.